### ISSUE TITLE
Entrada Blog Parque Nacional

### ISSUE THREAD

**Issue by AlfGarcia** [CarlosLaraR/AdAptA-Lab_Lupinus#3]
Esto es un draft para preparar la entrada del blog del Parque Nacional de Sierra de Guadarrama

**Comment by CarlosLaraR** [comment]
Genial @AlfGarcia !! Quieres que creemos una sección en la pestaña "code" o lo hacemos desde aquí?

**Comment by AlfGarcia** [comment]
No se como será mejor. Mi idea sería como un manuscrito que pudieramos editar todos a la vez.

**Comment by CarlosLaraR** [comment]
Genial. Así lo hacemos. También lo puedes subir como un archivo de word por si la gente quiere usar control de cambios. Pero como es algo breve puede funcionar así

**Comment by CarlosLaraR** [comment]
No es buena idea usar este chat para trabajar el manuscrito. Mejor las carpetas dentro de la pestaña code

**Comment by txiriondo** [comment]
Tal vez sería mejor trabajando sobre versiones de Word. El texto sobre la ventana de code se queda horizontal y se va por la derecha de la ventana y además si alguien lo edita no sabemos qué es lo que se ha editado ni quien lo ha editado, no?

**Comment by txiriondo** [comment]
Lo siento, lo he cerrado sin querer...

**Comment by CarlosLaraR** [comment]
Sí. eso creo que es mejor. He creado una carpeta nueva dentro de manuscripts para que el primero que escriba suba ahí su versión en formato word. El texto de Alf lo he dejado como un "readme" de esa carpeta

**Comment by CarlosLaraR** [comment]
@AlfGarcia he leído tus comentarios sobre la nota para el blog. Me gusta el título 3 (y el 2 también ) y yo lo centraría más en el punto 2 y 3. EL 1 mola pero es más general creo que el 2 y 3 se centran más en lo que hacemos y se puede escribir fácil. Es como hacer un resumen del artículo de ecosistemas y el del Plant Biology.

**Comment by AlfGarcia** [comment]
@CarlosLaraR @txiriondo @javimorente @samuelonbio He dejado un primer archivo con la parte más lúdica. A ver que os parece y con vuestros comentarios sigo escribiendo algo más decente.

Ale, a ver que tal se nos da esto

Alf

**Comment by javimorente** [comment]
Muy chulo!
Esto y con unas fotos guapas y creo que puede molar.
Un abrazo!

**Comment by CarlosLaraR** [comment]
Hola  a todos,
@AlfGarcia me ha gustado la nota. Te dejo algunas sugerencias en https://github.com/CarlosLaraR/AdAptA-Lab_Lupinus/blob/master/Manuscripts/blog_PN/Articulo%20Blog%20PN_CLR.docx

**Comment by txiriondo** [comment]
Hola, está quedando muy bien! Yo también he puesto mi aportación en https://github.com/CarlosLaraR/AdAptA-Lab_Lupinus/blob/master/Manuscripts/blog_PN/Articulo%20Blog%20PN_CLR_tx.docx

**Comment by samuelonbio** [comment]
Hola a todos,


No he contestado antes porque intente entrar en el github hace unos de días, no supe y se me olvido totalmente. No aporto ningun comentario porque me gusta la nota y creo que esta bien.



Un abrazo,

Samu

________________________________
De: txema iriondo <notifications@github.com>
Enviado: miércoles, 28 de junio de 2017 18:51:01
Para: CarlosLaraR/AdAptA-Lab_Lupinus
Cc: Samuel Prieto Benítez; Mention
Asunto: Re: [CarlosLaraR/AdAptA-Lab_Lupinus] Entrada Blog Parque Nacional (#3)


Hola, está quedando muy bien! Yo también he puesto mi aportación en https://github.com/CarlosLaraR/AdAptA-Lab_Lupinus/blob/master/Manuscripts/blog_PN/Articulo%20Blog%20PN_CLR_tx.docx
